Compilers.slnf

```diff
@@ -81,4 +81,4 @@
       "src\\Workspaces\\VisualBasic\\Portable\\Microsoft.CodeAnalysis.VisualBasic.Workspaces.vbproj"
     ]
   }
-}
+}
```

Comment:
was this change just a sort?

Reply:
Appears to have been by solution explorer.